tests/bud.bats

```diff
@@ -65,7 +65,7 @@ load helpers
   expect_line_count 8
   run_buildah bud --signature-policy ${TESTSDIR}/policy.json --layers -t test2 ${TESTDIR}/use-layers
   run_buildah --log-level=error images -a
-  expect_line_count 9
+  expect_line_count 10 
```

Comment:
I'm completely lost as to why the image count changes here, or why it changed in 4cf37c27. Do you understand it?

Reply:
No I didn't chew on that, but like that's it's reverted.  If I get time today will see if I can figure it out.

Reply:
No need: time is finite, and this is a reversion to how it was before, and tests are passing. Plus, the world is always better when there's a little mystery. Thanks again @TomSweeneyRedHat